modules/host/net.go

```diff
@@ -0,0 +1,49 @@
+package host
+
+import (
+	"net"
+
+	"github.com/NebulousLabs/Sia/encoding"
+)
+
+type rpcID [8]byte
```

Comment:
does it make sense to use the types package 'Identifier' type? Maybe it doesn't, just a thought.